diff --git a/src/lgfx/v1/platforms/esp32/common.cpp b/src/lgfx/v1/platforms/esp32/common.cpp
index 1ed66a3..7f94c54 100644
--- a/src/lgfx/v1/platforms/esp32/common.cpp
+++ b/src/lgfx/v1/platforms/esp32/common.cpp
@@ -807,7 +807,7 @@ namespace lgfx
  #if (ESP_IDF_VERSION >= ESP_IDF_VERSION_VAL(5, 0, 2) && ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 1, 0))
       (&dev->comd[0])[index].val = cmd_val;
  #else
-      (&dev->comd0)[index].val = cmd_val;
+      (&dev->comd[0])[index].val = cmd_val;
  #endif
 #else
       dev->command[index].val = cmd_val;
